design/user_types.go

```diff
@@ -457,3 +522,10 @@ var RelationWorkItemData = a.Type("RelationWorkItemData", func() {
 	})
 	a.Required("type", "id")
 })
+
+// WorkItemLinks has `self` as of now according to http://jsonapi.org/format/#fetching-resources
+var WorkItemLinks = a.Type("WorkItemLinks", func() {
```

Comment:
This name might be misleading because work item links are the once that I implemented in PR #421